rosjava/src/main/java/org/ros/internal/node/server/SlaveServer.java

```diff
@@ -52,12 +53,15 @@
   private final TopicParticipantManager topicParticipantManager;
   private final ParameterManager parameterManager;
   private final TcpRosServer tcpRosServer;
+  private final DefaultNode defaultNode;
```

Comment:
I'd suggest changing to `Node` instead of tying this to a particular implementation.
As we only need the `shutdown` method, we can use the base interface instead.